src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -94,6 +94,12 @@ public class AgentKnobs
             new EnvironmentKnobSource("AGENT_USE_NODE10"),
             new BuiltInDefaultKnobSource("false"));
 
+        public static readonly Knob HashValidationDisabled = new Knob(
+            nameof(HashValidationDisabled),
+            "If true, agent package hash validation during self-updating will be disabled.",
+            new EnvironmentKnobSource("AZP_AGENT_HASH_VALIDATION_DISABLED"),
+            new BuiltInDefaultKnobSource("false"));
```

Comment:
I think this naming scheme would be more consistent with other blobs:

```suggestion
        public static readonly Knob DisableHashValidation = new Knob(
            nameof(DisableHashValidation),
            "If true, agent package hash validation during self-updating will be disabled.",
            new EnvironmentKnobSource("AZP_AGENT_DISABLE_HASH_VALIDATION"),
            new BuiltInDefaultKnobSource("false"));
```

Reply:
Thanks, and let me also make related changes in the self-updater and move this Knob to the Misc section since this is a more appropriate location for it.